section-04-proj2/assets/scripts/app.js

```diff
@@ -0,0 +1,270 @@
+const ATTACK_VALUE = 10;
+const STRONG_ATTACK_VALUE = 17;
+const MONSTER_ATTACK_VALUE = 14;
+const HEAL_VALUE = 20;
+
+const MODE_ATTACK = 'ATTACK'; // MODE_ATTACK = 0
+const MODE_STRONG_ATTACK = 'STRONG_ATTACK'; // MODE_STRONG_ATTACK = 1
+const LOG_EVENT_PLAYER_ATTACK = 'PLAYER_ATTACK';
+const LOG_EVENT_PLAYER_STRONG_ATTACK = 'PLAYER_STRONG_ATTACK';
+const LOG_EVENT_MONSTER_ATTACK = 'MONSTER_ATTACK';
+const LOG_EVENT_PLAYER_HEAL = 'PLAYER_HEAL';
+const LOG_EVENT_GAME_OVER = 'GAME_OVER';
+
+const enterredValue = prompt('Max life for you and monster','100');
+
+let chosenAndreLife = parseInt(enterredValue);
+let battleLog = [];
+let lastLoggedEntery;
+
+if (isNaN(chosenAndreLife) || chosenAndreLife <= 0) {
+  chosenAndreLife = 100;
+}
+
+let currentMonsterHealth = chosenAndreLife;
+let currentPlayerHealth = chosenAndreLife;
+let hasBonusLife = true;
+
+adjustHealthBars (chosenAndreLife);
+
+function writeToLog (event, value, monsterHealth, playerHealth){
```

Comment:
JFYI (just for your information)

More than 3 arguments at a function is bad practice.